gravitee-apim-rest-api/gravitee-apim-rest-api-portal/gravitee-apim-rest-api-portal-rest/src/main/java/io/gravitee/rest/api/portal/rest/resource/SubscriptionsResource.java

```diff
@@ -104,6 +119,9 @@ public Response createSubscription(@Valid @NotNull(message = "Input must not be
                 );
                 newSubscriptionEntity.setGeneralConditionsContentRevision(generalConditionsContentRevision);
             }
+            newSubscriptionEntity.setFilter(subscriptionInput.getFilter());
+            newSubscriptionEntity.setMetadata(subscriptionInput.getMetadata());
+            newSubscriptionEntity.setConfiguration(MAPPER.valueToTree(subscriptionInput.getConfiguration()));
```

Comment:
@marcambier this one is for you :)

Reply:
The subscriptionInput is a openapi generated object, so we can't handle custom (de)serialization in it, like we would have done with jackson annotations on MAPI side.
So it ends with an Object type in the subscriptionInput, we have to convert here. It's a serialization to string, so AFAIK it can't be any 'bad format issue' here.
The subscription string consistency will be checked by service according to entrypoint type.